docs/aspenmesh-collector.MD

```diff
@@ -4,6 +4,14 @@ The Aspen OpenTelemetry Collector collects telemetry data from Kubernetes and yo
 
 ## Installation
 
+### One-line script installation
+
+The Aspen OpenTelemetry Collector can be installed by a one-line script.
```

Comment:
I vaguely remember Joanna saying something about us removing "OpenTelemetry" from our collector name? let's check, and do that here if needed